site/tryit/README.md

```diff
@@ -20,6 +20,10 @@ docker run \
 sorintlab/agolademo \
 serve --embedded-etcd --components all-base,executor
 ```
+where:
+   * `-v=path/to/local/agola/data` is your agola data directory
+   * `-p 8000:8000` is the agola port provisioned
```

Comment:
I'm not sure this last line really needs an explanation, it's just basic docker container port publishing.